projects/common/src/telemetry/user-telemetry-helper.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable, Injector } from '@angular/core';
+import { NavigationEnd, Router } from '@angular/router';
+import { filter } from 'rxjs/operators';
+import { Dictionary } from '../utilities/types/types';
+import { UserTelemetryProvider, UserTelemetryRegistrationConfig, UserTraits } from './telemetry';
+
+@Injectable({ providedIn: 'root' })
+export class UserTelemetryHelperService {
+  private telemetryProviders: UserTelemetryInternalConfig[] = [];
+
+  public constructor(private readonly injector: Injector, private readonly router: Router) {
+    this.setupAutomaticPageTracking();
+  }
+
+  public register(...configs: UserTelemetryRegistrationConfig<unknown>[]): void {
+    try {
+      const providers = configs.map(config => this.buildTelemetryProvider(config));
+      this.telemetryProviders = [...this.telemetryProviders, ...providers];
+    } catch (error) {
+      /**
```

Comment:
don't break, but also we should be logging this.

Reply:
Are you suggesting console error?

Reply:
or the logger service, but we don't use that much any more.

Reply:
yeah, that's why I didn't want to create a new dependency.

Reply:
if you don't want to depend on logger service use console.error, either way. Right now, there's no way to see a miconfiguration without either seeing it on the third party side or putting in a breakpoint.